pkgs/development/ocaml-modules/pure-splitmix/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildDunePackage }:
+
+buildDunePackage rec {
+  pname = "pure-splitmix";
+  version = "0.3";
+
+  src = final.fetchFromGitHub {
```

Comment:
```suggestion
  src = fetchFromGitHub {
```

Reply:
Sorry, this was copied from my overlays :sweat_smile: